tests/integration/test_base.py

```diff
@@ -48,8 +48,8 @@ def _add_scenario(self, identifier, title, xml):
         self.addCleanup(scenarios.remove_scenario, identifier)
 
     def _get_items(self):
-        items_container = self._page.find_element_by_css_selector('ul.items')
-        return items_container.find_elements_by_css_selector('li.option')
+        items_container = self._page.find_element_by_css_selector('div.items')
+        return items_container.find_elements_by_css_selector('div.option')
```

Comment:
Maybe a good occasion to not select based on the DOM tag name, but simply on the class?